### PR TITLE
sdk: python: Remove wrapping result object

### PR DESCRIPTION
**Pull request by helderco**
Related:
- #3555 
- #3692

Signed-off-by: Helder Correia

### CODE REVIEW

**Comment by gerhard** [comment]
I just started looking at this.

File: sdk/python/README.md [hunk 1]
```diff
@@ -1,11 +1,74 @@
 # Dagger Python SDK
 
-Install [hatch](https://hatch.pypa.io/latest/install/). Recommend [`pipx`](https://github.com/pypa/pipx)), e.g. `pipx install hatch`
+A client package for running [Dagger](https://dagger.io/) pipelines.
 
-Run tests with `hatch run test`. Assumes `go run ./cmd/cloak dev --workdir sdk/python` is running.
```
Comment (gerhard) [inline sdk/python/README.md:5]: `hatch run test` will fail without this. I would only remove it when that is no longer the case.

File: sdk/python/README.md [hunk 2]
```diff
@@ -1,11 +1,74 @@
...
+| Simple is better than complex |
+  =============================
+                             \
+                              \
+                                ^__^
+                                (oo)\_______
+                                (__)\       )\/\
+                                    ||----w |
+                                    ||     ||
+```
```
Comment (gerhard) [inline sdk/python/README.md:56]: I followed this example and it failed as follows:

image

Could we add what I missed to this README? It seems incomplete without it.

File: sdk/python/README.md (hunk 1 above)
Comment (helderco) [reply sdk/python/README.md:5]: I have a tangle of PRs, the fix to that is in https://github.com/dagger/dagger/pull/3716, I'd rather just push this on.

**Pull request merged**